scss/_buttons.scss

```diff
@@ -190,14 +190,17 @@ fieldset:disabled a.btn {
     }
 
     &,
-    &.btn-inverse:active {
+    &.btn-inverse:active,
+    &.btn-inverse.active {
       background-color: $white;
       background-image: escape-svg(url("data:image/svg+xml,#{$icon}"));
+      border-color: $white;
```

Comment:
This line should maybe be in another selector since it breaks the normal social buttons